modules/e2ee/E2EEncryption.js

```diff
@@ -96,10 +32,13 @@ export class E2EEncryption {
      * @returns {boolean}
      */
     static isSupported(config) {
+        if (!config.externallyManagede2ee && !OlmAdapter.isSupported()) {
```

Comment:
Maybe we should create a dedicated "e2ee" section instead. With a "mode" option.